test/integration/BUILD

```diff
@@ -28,8 +28,8 @@ skiko_runtime_platform = select({
      "//conditions:default": ["INVALID"]})
 
 compose_test_libraries = [
-    "test-integration",
-    "test-utils",
+    "//test/integration/common:common",
+    "//test/integration:test-integration",
```

Comment:
We're using absolute paths now in line with other Studio BUILD files as we aren't within a single folder anymore.